build_scripts/build_usd.py

```diff
@@ -1465,14 +1840,64 @@ def InstallOpenSubdiv(context, force, buildArgs):
         # just 1 job for now. See:
         # https://github.com/PixarAnimationStudios/OpenSubdiv/issues/1194
         oldNumJobs = context.numJobs
+        extraEnv = os.environ.copy()
+        buildDirmacOS = ""
         if MacOS():
             context.numJobs = 1
 
+        if context.targetIos:
```

Comment:
@davidgyu Would it be possible to integrate these changes upstream at OpenSubdiv? Once again, we can point to a later version of OpenSubdiv for iOS in preference to patching OpenSubdiv.

Reply:
We've already fixed most (if not all) of these concerns upstream as of the current OpenSubdiv 3.5 release.
- We no longer need the numJobs override above.
- We no longer need a macOS dependent build for STRINGIFY.
- We should no longer need to extend RunCMake() with envOveride or hostPlatform optional args.

Reply:
Oh, fantastic :) So pulling that version, at least for iOS, and identifying any remaining issues seems like it is the way forward.

Reply:
Simplified all the things David mentioned!